specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-10-01-preview/mfe.json

```diff
@@ -9,6 +9,94 @@
     "https"
   ],
   "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.MachineLearningServices/availableQuota": {
+      "get": {
+        "tags": [
+          "Quota"
+        ],
+        "summary": "Get available MaaS PTU quota.",
+        "operationId": "PTUQuota_GetAvailable",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "subscriptionId",
+            "required": true,
+            "type": "string",
+            "description": "The ID of the target subscription."
+          },
```

Comment:
Any reason why you're not using the common type?

Reply:
Not intended, we should be using common type.